Documentation/CRDs/ceph-cluster-crd.md

```diff
@@ -1339,6 +1339,7 @@ In order to configure an external Ceph cluster with Rook, we need to extract som
 - `--rbd-data-pool-name`: The name of the RBD data pool
 - `--rbd-metadata-ec-pool-name`: (optional) Provides the name of erasure coded RBD metadata pool
 - `--rados-namespace`: (optional) Divides a pool into separate logical namespaces
```

Comment:
If `--rados-namespace` is already here, is this PR really just about subvolume groups?

Reply:
these PR is more about creating Storage class using subvolumes and rados namespace